onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -374,7 +374,10 @@ def __init__(
         self._sess_options = sess_options
         self._sess_options_initial = sess_options
         self._enable_fallback = True
-        self._read_config_from_model = os.environ.get("ORT_LOAD_CONFIG_FROM_MODEL") == "1"
+        if "read_config_from_model" in kwargs:
```

Comment:
is this documented somewhere? why "read" instead of "load" as in the existing environment variable name?

Reply:
True, if forgot to remove.